dockerfiles/Dockerfile.arch-freeze

```
@@ -0,0 +1,47 @@
+FROM scratch
+LABEL maintainer="Iterait a.s. <docker@iterait.com>"
+
+COPY --from=archlinux:latest / /
+# add cleanup hook
+ADD 99-cleanup.hook /usr/share/libalpm/hooks/
+
+
+# freeze Arch to specific date
+ARG FREEZE_DATE='2024/04/07'
```

I suggest dropping the default value

```suggestion
ARG FREEZE_DATE
```